hazelcast/src/main/java/com/hazelcast/map/impl/operation/MapFetchIndexOperation.java

```diff
@@ -406,7 +407,7 @@ public int getClassId() {
         }
     }
 
-    public static final class MissingPartitionException extends HazelcastException {
+    public static final class MissingPartitionException extends HazelcastException implements SilentException {
```

Comment:
This is not the purpose documented in `SilentException`... That interface is for exceptions that are sent to the user anyway, so there's no point in logging them on server. For example, query failures could be logged like this. And it has only couple of implementations. 

It seems it would be cleaner to override the `logError()` method in `MapFetchIndexOperation`...